test/e2e/fixtures/ngJq/script.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
```

Comment:
This line breaks jshint rules. It should either be made into the functional form, or perhaps easier just remove it.